azuredevops/azuredevops.go

```diff
@@ -129,6 +133,10 @@ func addOptions(s string, opt interface{}) (string, error) {
 		return s, err
 	}
 
+	for k, v := range u.Query() {
```

Comment:
Hi there @fatal10110 - Is there any chance you could add some unit tests for this please?

Reply:
Sure :)